soh/soh/GameMenuBar.cpp

```diff
@@ -1168,6 +1168,8 @@ namespace GameMenuBar {
                 }
                 UIWidgets::PaddedEnhancementCheckbox("Fix Gerudo Warrior's clothing colors", "gGerudoWarriorClothingFix", true, false);
                 UIWidgets::Tooltip("Prevent the Gerudo Warrior's clothes changing color when changing Link's tunic or using bombs in front of her");
+                UIWidgets::PaddedEnhancementCheckbox("Fix Camera Drift", "gFixCameraDrift", true, false);
+                UIWidgets::Tooltip("Fixes the math for sin/cos accuracy in calucaltation from spherical to cartisean cordinates");
```

Comment:
Ignoring the spelling mistakes, I'd like a tooltip that better explains to a play what this fix actually fixes. I've seen an example video of the previous behaviour but even for me it's not entirely clear what this fix does, let alone someone that sees this tooltip.

Reply:
beyond adding the word drift i really dont know what else you want...how am I supposed to say that there is a positive drift that happens from -1000 aprox to 0 rotation and the same from -17384 to -16384.. This Fixes the math for sin/cos accuracy in calculation from spherical to cartesian cordinates removing this positive drift...that's way too complex for a user to see...im sorry if you cant see the issue, but it exists non the less

Reply:
you knida need to realize that the drift is coming in from going back and forth between cartesian and spherical coordinates which is what the camera code does, if you have some better tool tip idea I'm open to constructive criticism

Reply:
the drift is the same concept as the vernier effect, or compounding err in inaccurate calculations, i really hope this helps you understand

Reply:
watch closely for the slow drift to the left:
https://1drv.ms/v/s!Ap9To4aDM22NgfoFeeqGga5svnM4TA?e=3zHJjI

Reply:
Something like "Fixes camera slightly drifting to the left when standing still due to a math error." 

The player generally cares more about what something means to them rather than the technological explanation of it.

Reply:
that sounds good to me